pages/common/ac.md

```diff
@@ -0,0 +1,11 @@
+# ac
+
+> A faster version of cat that can strip ANSI color codes.
+
+- Dump a file's contents:
+
+`ac file`
```

Comment:
Perhaps you could make file `{{path/to/file}}` - The `{{` and `}}` are our way of indicating that a user can alter a parameter.